src/style.css

```diff
@@ -0,0 +1,115 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  padding: 32px 14px 14px;
+  max-width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  margin: 0 5px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__name {
+  font-style: normal;
+  margin: 36px 2px 4px;
+  width: 166px;
+  color: #060b35;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+}
+
+.card__code {
+  margin: 0 2px 16px;
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  margin-left: 2px;
+}
+
+.stars__star {
+  margin-right: 4px;
+  background-image: url(./images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--4 .stars__star:not(:last-child) {
+  background: url(./images/star-active.svg) no-repeat center;
+}
+
+.card__stars {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 21px;
+}
+
+.card__reviews {
+  font-size: 10px;
+  line-height: 17px;
+  margin-right: 2px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 19px;
+}
+
+.card__price-text {
+  font-size: 12px;
+  line-height: 12px;
+  margin-left: 2px;
+  margin-top: 5px;
+}
+
+.card__price-numbers {
+  font-size: 16px;
+  font-weight: bold;
+  line-height: 12px;
+  margin-right: 2px;
+  margin-top: 5px;
+}
+
+.card__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 40px;
+  width: 166px;
+  font-weight: 700;
+  line-height: 16px;
+  font-size: 14px;
+  text-decoration: none;
+  color: #fff;
+  background-color: #00acdc;
+  border-radius: 5px;
+  border: 1px solid #00acdc;
+  margin-left: 2px;
+}
```

Comment:
organize your css properties according this [rules](https://9elements.com/css-rule-order/), fix it everywhere